tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1139,6 +1139,25 @@ public function theUserCreatesAFolderUsingTheGraphApi(
 		}
 	}
 
+	/**
+	 * @When /^user "([^"]*)" tries to create subfolder "([^"]*)" in the nonexistent folder of the space "([^"]*)" using the WebDav Api$/
+	 *
+	 * @param string $user
+	 * @param string $subfolder
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 *
+	 * @throws GuzzleException
+	 */
+	public function theUserTriesToCreatesASubFolderUsingTheGraphApi(
+		string $user,
+		string $subfolder,
+		string $spaceName
+	): void {
+		$this->theUserCreatesAFolderToAnotherOwnerSpaceUsingTheGraphApi($user, $subfolder, $spaceName);
```

Comment:
This existing function has a name that is a bit confusing - actually in this case it is a project space, which does not have an "owner". Alice is a manager of the space.

But that is existing code, so does not need to be touched for this PR!